addons/mail/static/src/discuss/call/common/rtc_session_model.js

```diff
@@ -62,8 +62,10 @@ export class RtcSession extends Record {
     /** @type {Date|undefined} */
     raisingHand;
     videoComponentCount = 0;
+    /** @type {Map<string, MediaStream>} */
+    videoStreams = new Map();
```

Comment:
naming convention for maps: \<value\>By\<key\>, so in our case, I guess it'd something like “videoStreamByName”.

But do you expect this map to contain anything other than screen and camera? If there are only these two streams, it would probably be simpler to just have them as separate fields on RtcSession, `screenStream` and `cameraStream` for example